src/Loading/sceneLoader.ts

```diff
@@ -123,7 +123,7 @@ export interface ISceneLoaderPluginBase {
      * @param data string containing the data
      * @returns data to pass to the plugin
      */
-    directLoad?(scene: Scene, data: string): any;
+    directLoad?(scene: Scene, data: string): Promise<any>;

```

Comment:
```suggestion
    directLoadAsync?(scene: Scene, data: string): Promise<any>;
```

I'm not sure this is okay to change, but typically, our async functions that return a Promise have a suffix of `Async`.

Reply:
@sebavan ?

Reply:
I would prefer to not change the signature if possible to prevent any regression

Reply:
It already is changing the signature, so there isn't really a difference.